### PR TITLE
B feature/pending status management

### PR DESCRIPTION
**Pull request by dujfalusi**

### CODE REVIEW

File: src/main/java/eu/execom/hawaii/service/AllowanceService.java [hunk 1]
```diff
@@ -39,11 +42,58 @@ Allowance getByUser(User user) {
...
+      allowanceRepository.save(allowance);
+    }
+
+  }
+
+  private void applyPendingAnnual(Allowance allowance, int requestedHours) {
+    var allowancePendingAnnual = allowance.getPendingAnnual();
+    var calculatedPendingAnnual = allowancePendingAnnual + requestedHours;
+    allowance.setPendingAnnual(calculatedPendingAnnual);
+    allowanceRepository.save(allowance);
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/service/AllowanceService.java]: This line is duplicated both in calling method. Please remove one of them.

File: src/main/java/eu/execom/hawaii/service/AllowanceService.java [hunk 2]
```diff
@@ -39,11 +42,58 @@ Allowance getByUser(User user) {
...
+  }
+
+  private void applyPendingTraining(Allowance allowance, int requestedHours) {
+    var allowancePendingTraining = allowance.getPendingTraining();
+    var calculatedPendingAnnual = allowancePendingTraining + requestedHours;
+    allowance.setTakenTraining(calculatedPendingAnnual);
+    allowanceRepository.save(allowance);
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/service/AllowanceService.java]: This line is duplicated both in calling method. Please remove one of them.

Comment (dujfalusi) [reply src/main/java/eu/execom/hawaii/service/AllowanceService.java]: removed

**Comment by stanislavgrujic** [comment]
Please resolve merge conflicts.

**Pull request merged**